lib/readers/json_file_reader.js

```diff
@@ -60,3 +61,15 @@ function toFilePath(entityName) {
   }
   return path.join('.jhipster', `${upperFirst(entityName)}.json`);
 }
+
+/**
+ * From an entity's name, gives the expected previous state file path.
+ * @param entityName the entity's name.
+ * @returns {string} the file's path.
+ */
+function toPreviousStateFilePath(entityName) {
```

Comment:
`getFilePathForPreviousStateOfEntity` would be a better name

Don't hesitate to be verbose!